Kudu.Core/SiteExtensions/SiteExtensionManager.cs

```diff
@@ -801,7 +856,18 @@ private void CleanupSiteExtensionJobs(string siteExtensionName)
 
         private async Task<SiteExtensionInfo> CheckRemotePackageLatestVersion(SiteExtensionInfo info, UIMetadataResource metadataResource)
         {
-            UIPackageMetadata localPackage = await metadataResource.GetLatestPackageByIdFromMetaRes(info.Id);
+            bool isNuGetPackage = false;
+
+            if (!string.IsNullOrEmpty(info.FeedUrl))
+            {
+                isNuGetPackage = System.Text.RegularExpressions.Regex.IsMatch(
```

Comment:
Why not use the helper that does that check to avoid duplication?

Reply:
Because I was too lazy to refactor IsNuGetRepo to take a string instead of a SourceRepository.  I'll fix it.